internal/operators/ocs/consts.go

```diff
@@ -3,10 +3,14 @@ package ocs
 type ocsDeploymentMode string
 
 const (
-	ssdDrive     string            = "SSD"
-	hddDrive     string            = "HDD"
-	minDiskSize  int64             = 5 //5GB is the min disk size for OCS
-	compactMode  ocsDeploymentMode = "Compact"
-	minimalMode  ocsDeploymentMode = "Minimal"
-	standardMode ocsDeploymentMode = "Standard"
+	CPUCompactMode    int64             = 6  // per host requires 8 cpus for OCS in compact mode
+	CPUMinimalMode    int64             = 4  // per host requires at least 6 cpus for OCS in minimal mode
+	MemoryCompactMode int64             = 19 // Memory value provided in GB for per host
```

Comment:
Have you considered adding units as part of the variable name? Like it is being done [here](https://github.com/openshift/assisted-service/blob/master/internal/hardware/validator.go#L43-L45)
```
	MinRamGib                        int64                        `envconfig:"HW_VALIDATOR_MIN_RAM_GIB" default:"8"`
	MinRamGibWorker                  int64                        `envconfig:"HW_VALIDATOR_MIN_RAM_GIB_WORKER" default:"8"`
	MinRamGibMaster                  int64                        `envconfig:"HW_VALIDATOR_MIN_RAM_GIB_MASTER" default:"16"`
```

It might be better to know what sort of units is used in this variable when referencing it elsewhere in the code, such to avoid using the wrong units.